config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   root 'users#index'
   resources :users, only:[:index, :show] do
-    resources :posts, only:[:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:create]
```

Comment:
- [ ] There should be a pair of routes (create and new )in the config/routes.rb file for creating comments. Kindly let's add this. ✍🏻